models/simple_hearing_aid/readme.md

```diff
@@ -0,0 +1,31 @@
+# A Simple Hearing Aid
+This simple hearing aid example is a combination of the [Adaptive Wiener Filter Noise Suppression Model](../noise_suppression) and the [FFT Filter Model](../fft_filters).
+
```

Comment:
Add `This model was developed using the Frost Autogen Framework and deployed to an Audio Mini using Frost Edge. To set up Frost Autogen or Frost Edge, please review the [Getting Started Guides](https://github.com/fpga-open-speech-tools/docs/tree/master/getting_started). `

Reply:
Done with 3ec81dd